app/Models/BookRepository.php

```
@@ -959,6 +985,47 @@ private function syncAuthors(int $bookId, array $authorIds): void
+     */
+    private function syncPublishers(int $bookId, array $publisherIds): void
+    {
+        $stmt = $this->db->prepare('DELETE FROM libri_editori WHERE libro_id = ?');
+        if ($stmt) {
+            $stmt->bind_param('i', $bookId);
+            $stmt->execute();
+            $stmt->close();
+        } else {
+            error_log("Critical error: Unable to prepare statement for deleting book publishers for book_id: $bookId");
+            throw new \Exception("Database error: unable to delete book publishers");
+        }
```

PHPMD (2.15.0)

[error] The method syncPublishers uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)